beginner_source/nlp/advanced_tutorial.py

```diff
@@ -177,6 +177,12 @@ def __init__(self, vocab_size, tag_to_ix, embedding_dim, hidden_dim):
         self.transitions = nn.Parameter(
             torch.randn(self.tagset_size, self.tagset_size))
 
+        # These two statements enforce the constraint that we never transfer *to* the start tag,\n",
+        # and we never transfer *from* the stop tag (the model would probably learn this anyway,\n",
+        # so this enforcement is likely unimportant)\n",
+        self.transitions.data[tag_to_ix[START_TAG], :] = -10000\n",
```

Comment:
What are the `\n",`  at the end of the lines?
Can you also please limit line length to 80.